File: 백준/Bronze/10951. A＋B － 4/A＋B － 4.cs
using System;

class Program
{
    static void Main()
    {
        string input;
        while ((input = Console.ReadLine()) != null)
        {
            string[] numbers = input.Split();
            if (numbers.Length == 2 && int.TryParse(numbers[0], out int a) && int.TryParse(numbers[1], out int b))
            {
                Console.WriteLine(a + b);
            }
        }
    }
}

File: 백준/Bronze/10951. A＋B － 4/A＋B － 4.js
const input = require('fs').readFileSync('/dev/stdin').toString().split('\n');

for (let line of input) {
    line = line.trim();
    if (line) {
        let numbers = line.split(' ');
        if (numbers.length === 2) {
            let a = parseInt(numbers[0]);
            let b = parseInt(numbers[1]);
            if (!isNaN(a) && !isNaN(b))
                console.log(a + b);
        }
    }
}